lib/views/widgets/tabs/tab_layout.dart

```diff
@@ -1,14 +1,16 @@
 import 'package:flutter/material.dart';
 import 'package:sun_flutter_capstone/views/widgets/tabs/tab_buttons.dart';
 
+// Refactor receiving of tab details if there's time
```

Comment:
To easily search TODO comments later
```suggestion
// TODO: Refactor receiving of tab details if there's time
```
```suggestion
// Refactor receiving of tab details if there's time
```